src/blog-post-workflow.js

```diff
@@ -140,7 +143,7 @@ feedList.forEach((siteUrl) => {
               if (ENABLE_SORT && ENABLE_VALIDATION && !item.pubDate) {
                 reject('Cannot read response->item->pubDate');
               }
-              
```

Comment:
revert unnecessary style changes like this

Reply:
I understand your gut reaction, but these are mere white space deletions. These shouldn't be here in the first place.